Build issue: Make readdir, closedir, opendir static in nob.h

```
raylib.lib(rcore.obj) : error LNK2005: opendir already defined in plug.obj
raylib.lib(rcore.obj) : error LNK2005: closedir already defined in plug.obj
raylib.lib(rcore.obj) : error LNK2005: readdir already defined in plug.obj
```
Both rcore.c and plug.c (via nob.h) export opendir, closedir and readdir. This results in a linker error. I fixed it currently by declaring them static in nob.h.

--- a/src/nob.h
+++ b/src/nob.h
@@ -364,9 +364,9 @@ struct dirent
 
 typedef struct DIR DIR;
 
-DIR *opendir(const char *dirpath);
-struct dirent *readdir(DIR *dirp);
-int closedir(DIR *dirp);
+static DIR *opendir(const char *dirpath);
+static struct dirent *readdir(DIR *dirp);
+static int closedir(DIR *dirp);
 #endif // _WIN32
 // minirent.h HEADER END ////////////////////////////////////////
 

